src/v/kafka/server/group.cc

```diff
@@ -1875,16 +1874,28 @@ group::prepare_tx(cluster::prepare_group_tx_request r) {
       raft::replicate_options(raft::consistency_level::quorum_ack));
 
     if (!e) {
+        // Prev logic delete tx from volatile before replication. Sp we should
```

Comment:
We may drop this comment because it more explains a change in respect to the previous version rather that what it does.